### PR TITLE
add more heterogeneous cluster ci jobs

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-azure-ipi-disconnected-fullyprivate-p2-f14

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-azure-ipi-disconnected-fullyprivate-p2-f14

**Comment by jianlinliu** [comment]
cc @aleskandro and @liangxia to review.

**Review by aleskandro — APPROVED** [review]

**Review by aleskandro — COMMENTED** [review]
/hold

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 1]
```diff
@@ -79,6 +79,38 @@ tests:
...
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@heterogeneous and @azure-ipi and @network-ovnkubernetes and
+        not @fips'
+      TAG_VERSION: '@4.12'
+    test:
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
+- as: azure-ipi-disconnected-fullyprivate-p2-f14
```
Comment (aleskandro) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:98]: @jianlinliu remove this as not supported on 4.12.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:98]: I saw we have a ci profile - `04_heterogeneous_Disconnected IPI on Azure & fully Private Cluster & OVN & x86_arm64_workers`, is that true we dose not support it?

Comment (aleskandro) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:98]: yes, it is. That was an in-advance testing done to gather some infos.

The blocking aspect of disconnected on heterogeneous clusters is the support for sparse manifest lists in order to allow the customers to mirror only a subset of the manifests (the ones for the arches they actually want).

However, as that effort seems still far from landing in production, we might want to enable disconnected support for 4.14 as is (users will need to mirror the whole payload as we usually do for testing now). 

Similar concerns are being tracked for the operators catalog that will increase a lot the magnitude of the required storage to support multi-arch clusters.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:98]: Thanks for the explaination.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml [hunk 2]
```diff
@@ -95,6 +95,22 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
+- as: azure-ipi-disconnected-fullyprivate-p2-f14
```
[the remark this answers is not in the document]
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml]: fixed now.

**Review by aleskandro — APPROVED** [review]
Thanks @jianlinliu !

**Comment by jianlinliu** [comment]
rebase again to make required test jobs get passed.

**Pull request merged**